tests/Model/ApnsMessageTest.php

```diff
@@ -625,17 +626,100 @@ public function testGetPayloadWithMultipleTokensAndFullMessageSet()
                             'something' => 'something',
                         ),
                         'launch-image' => 'something',
+                        'action' => array(
+                            array(
+                                'something' => 'something',
+                            )
+                        )
```

Comment:
add a comma here

as you can see, the last line of the array before you add your code (`'launch-image' => 'something',`) is not affect by the diff, because you did not have to add a comma at the end of the line... because the comma was already there :)